### DISCUSSION TITLE
Прохождение без гильдии

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#355] [Ideas]
Какие враги при игре за нейтрала появятся в 4 главе?

-У Мага появляются Ищущие
-У ОнД люди-ящеры
-У Пала Орки Командиры
-У Нейтрала

Смешано? Всех по немного? Или может ищущие? 

Или  может сделать много одержимых, при чем одержимых усилить по параметрам, что бы были сильными противниками?

**Comment by Kor-Angar** [comment]
еще надо подумать про амулет который появляется во 2 главе. такой что бы полезный был. например повышал и силу и ловкость

**Comment by Kor-Angar** [comment]
так же у меня была мысль про воровство записки у Декстера. её можно сделать для игры нейтралом.

**Comment by Kor-Angar** [comment]
вот что я нашёл у себя 
![image](https://user-images.githubusercontent.com/49514764/64922274-22e64400-d7d6-11e9-958b-31a324276c68.png)

![image](https://user-images.githubusercontent.com/49514764/64922288-55903c80-d7d6-11e9-99c0-9ca3c8d4806e.png)

это я делал для заставки 3 главы
https://yadi.sk/d/b1uQ9a53-xOKnA

**Comment by dosinabox** [comment]
Круто. Ещё нужна отдельная заставка для пятой главы. Но уже в других доспехах, новых :)

**Comment by Kor-Angar** [comment]
а в каких герой будет доспехах в пятой главе? :-)

**Comment by dosinabox** [comment]
Охотника на орков) itar_oht_l + ithe_oht.
Характеристики придумать надо.

**Comment by Kor-Angar** [comment]
А откуда он их возьмёт и в какой главе?

**Comment by Kor-Angar** [comment]
А какое у него будет оружие? Там же драконы говорят что оружие специально для тебя что бы убить главного дракона. И доспехи должны быть тоже. А доспехов должно быть два, одни к концу 3 главы, а вторые в 5.

**Comment by dosinabox** [comment]
Давай ты продумаешь, а я сделаю) Я в таких тонкостях не особо разбираюсь.

**Comment by Kor-Angar** [comment]
тут же дело вовсе не в доспехах на самом деле. дело тут в том что я описывал в первом посте.

Доспехи герою разные можно сделать. можно даже сделать на выбор разные, например:
в 3 главе дать возможность получить доспехи Рудниково Борона после убийства Равена
если не получил (пропустил)
в 4 главе в Башне Ксардаса  "Доспехи Охотника на Демонов"
если не получил (пропустил)
в 4 главе доспехи Тамплиера от Кор Ангара
если не получил (пропустил)
в 4 главе от Ур-Шака "Доспехи Охотника на Орков"

в 5 главе можно дать герою доспехи Зодчих(в которых был Равен) или что то другое

в альманахе можно разместить рецепт зелья которое герой сможет приготовить например из слез инноса и крови дракона на увеличение основного параметра (силы или ловкости). ну или придумывать какое то оружие, что на мой взгляд не очень то хорошо, так как это не мод, а просто возможность пройти без гильдии.

**Comment by Kor-Angar** [comment]
Можно для нейтрала добавить Ищущих в 4 главе, а им в аммуницию добавить альманахи и сделать квест на поиск альманахов (штук 13).  Добавлять их не по одному, а группами по 4. Тогда они будут жесткими противниками 🙂 альманахи ищущих так же сдавать Пирокару.

**Comment by Kor-Angar** [comment]
добавить для нейтрала покупку чешуи Беннетом

при получение всей чешуи, возможность купить доспех или улучшить доспех или что то в этом духе.

**Comment by Kor-Angar** [comment]
подскажи пожалуйста, а как сделать проверку в диалоге на то сколько у  ГГ защиты от огня? 

	protection [PROT_FIRE] >50

**Comment by dosinabox** [comment]
В диалогах:
```
if(other.protection[PROT_FIRE] > 50)
```

Вне диалогов:
```
if(hero.protection[PROT_FIRE] > 50)
```

**Comment by Kor-Angar** [comment]
Спасибо18:18, 23 сентября 2020 г., dosinabox <notifications@github.com>:
В диалогах:
if(other.protection[PROT_FIRE] > 50)
Вне диалогов:
if(hero.protection[PROT_FIRE] > 50)

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by Kor-Angar** [comment]
я думаю нейтралу прям по поводу брони можно не замарачиваться. Ксардас там говорил про защиту от Дракона, поэтому можно просто сделать проверку на защиту от огня например 50, а там дальше посмотреть сможет игрок набрать через доспех из ползунов + кольца, амулеты и прочее, значит пройдёт, а если не сможет тогда придётся героя отправится в Ярик за доспехами :-)

```
func void DIA_Xardas_RITUALREQUEST_Info()
{
	AI_Output(other,self,"DIA_Xardas_RITUALREQUEST_15_00");	//Ватрас отправил меня к тебе.
	AI_Output(self,other,"DIA_Xardas_RITUALREQUEST_14_01");	//Хорошо. Что он сказал?
	AI_Output(other,self,"DIA_Xardas_RITUALREQUEST_15_02");	//Он сказал что-то о ритуале обращения в Круге Солнца.
	AI_Output(self,other,"DIA_Xardas_RITUALREQUEST_14_03");	//(смеется) Вот старый дьявол. Думаю, я знаю, что он затеял. Ты пришел, чтобы призвать меня к нему.
	AI_Output(other,self,"DIA_Xardas_RITUALREQUEST_15_04");	//Похоже на то. Когда ты отправляешься?
	
	if (hero.guild == GIL_KDF)
	|| (hero.guild == GIL_DJG)
	|| (hero.guild == GIL_PAL)
	|| ((GuildlessMode == TRUE) && (other.protection[PROT_FIRE] > 49))
	{
		AI_Output(self,other,"DIA_Xardas_RITUALREQUEST_14_05");	//Не стоит заставлять Ватраса ждать. Я отправляюсь немедленно. А ты должен выполнить свою задачу, а затем присоединиться ко мне опять.
		AI_StopProcessInfos(self);
		if(C_BodyStateContains(self,BS_SIT))
		{
			AI_UseMob(self,"THRONE",-1);
		};
		B_LogEntry(TOPIC_INNOSEYE,"Ксардас согласился участвовать в ритуале в Круге Солнца.");
		B_GivePlayerXP(XP_AmbientKap3 * 2);
		Npc_ExchangeRoutine(self,"RitualInnosEyeRepair");
		Xardas_GoesToRitualInnosEye = TRUE;
	}
	else
	{
		AI_Output(self,other,"DIA_Xardas_RITUALREQUEST_14_06");	//Я буду помогать в этом ритуале только когда буду уверен, что ты готов к встрече с драконами.
		B_GivePlayerXP(XP_AmbientKap3);
	};
};
```
```

///////////////////////////////////////////////////////////////////////
//	Info bereit
///////////////////////////////////////////////////////////////////////
instance DIA_Xardas_BEREIT(C_Info)
{
	npc = NONE_100_Xardas;
	condition = DIA_Xardas_BEREIT_Condition;
	information = DIA_Xardas_BEREIT_Info;
	description = "Я готов к сражению с драконами.";
};


func int DIA_Xardas_BEREIT_Condition()
{

	if  (Xardas_GoesToRitualInnosEye == FALSE)
	&& 	(Npc_KnowsInfo(other, DIA_Xardas_RITUALREQUEST))	
	&&	((hero.guild == GIL_DJG)
	||   (hero.guild == GIL_PAL))
	{
		return TRUE;
	};
	if	(Npc_KnowsInfo(other,DIA_Xardas_RITUALREQUEST)) 
	&&	(Xardas_GoesToRitualInnosEye == FALSE) 
	&&	(GuildlessMode == TRUE) 
	&&	(other.protection[PROT_FIRE] > 49)
	{
		return TRUE;
	};
};
```

**Comment by Kor-Angar** [comment]
в 3 главе бандиты появляются при игре за наёмника можно добавить их появление так же и нейтралу


у Торуса и Горна голоса одинаковые?

**Comment by dosinabox** [comment]
> у Торуса и Горна голоса одинаковые?

да

**Comment by Kor-Angar** [comment]
может тогда как то так

```
AI_Output(other,self,"DIA_Addon_BDT_10014_Thorus_Raventot_15_05");	//Кто знает, может, наши пути еще пересекутся.
if(hero.guild == GIL_None)
{	
	AI_Output(self,other,"DIA_GornOW_Add_12_00");	//Подожди, у меня есть кое-что, что могло бы пригодиться тебе.
	AI_Output(self,other,"DIA_GornOW_Add_12_02");	//Для меня они слишком малы, но тебе могут подойти...
	CreateInvItem(hero,ITAR_G1_Ebr);
}
else
{
	AI_Output(self,other,"DIA_Addon_BDT_10014_Thorus_Raventot_12_06");	//Кто знает. У тебя на пути будет много ворот и много переходов. И на одном из них ты можешь встретить меня...
};
```

**Comment by dosinabox** [comment]
```
AI_PrintScreen("Доспехи барона получено",-1,YPOS_ItemTaken,FONT_ScreenSmall,2);
```

**Comment by Kor-Angar** [comment]
> Какие враги при игре за нейтрала появятся в 4 главе?
> 
> -У Мага появляются Ищущие
> -У ОнД люди-ящеры
> -У Пала Орки Командиры
> -У Нейтрала
> 
> Смешано? Всех по немного? Или может ищущие?
> 
> Или может сделать много одержимых, при чем одержимых усилить по параметрам, что бы были сильными противниками?

ищущие. они более жёсткие но при этом не является фракционным квестом.
https://github.com/Kor-Angar/G2_NotR_Kor_Angar/commit/011cb9ddff97790786455ccbe3b3f9a16628ed80

**Comment by dosinabox** [comment]
А если на момент начала 4 главы ГГ будет послушником?

**Comment by Kor-Angar** [comment]
послушника мне кажется на церемонии восстановления глаза инноса или принимать в маги огня или лепить так новую гильдию про убийцу демонов. можно пока просто принять в маги огня.

**Comment by Kor-Angar** [comment]
не могу найти. у меня по послушнику было текста страницы на 2 А4. там были и доп плюшки в виде свитков во второй главе от Мильтена в Минентале и эксперсс приём в Маги Огня от Пирокара с бесплатным первым кругом и парой рун. а так же был альтернативный вариант приёма из Послушники в Паладины.

**Comment by Kor-Angar** [comment]
у Торуса и Горна  в русской версии [голоса разные](https://www.youtube.com/watch?v=o_hxKxuDgwk) :-)

**Comment by Kor-Angar** [comment]
наверное и квест про Salandrilа надо давать нейтралу

```
		else if((hero.guild == GIL_SLD) || (hero.guild == GIL_DJG)|| (hero.guild == GIL_NONE))
		{
			if(!Npc_IsDead(Salandril))
			{
				return TRUE;
			};
		};
```

**Comment by Kor-Angar** [comment]
квест Дара

```
func int DIA_Dar_BRINGORCELITERING_Condition()
{
	if((MIS_Dar_BringOrcEliteRing == LOG_Running) 
	//&& ((hero.guild == GIL_SLD) || (hero.guild == GIL_DJG) || (hero.guild == GIL_KDF)) 
	&& Npc_HasItems(other,ItRi_OrcEliteRing))
	{
		return TRUE;
	};
};
```

**Comment by Kor-Angar** [comment]
Жан в оригинале тоже делает доспехи нейтралу?

**Comment by Kor-Angar** [comment]
чешую забирает, а доспехи не продаёт

**Comment by Kor-Angar** [comment]
диалог Сильвио надо переделать. проверку сделать на фразу для паладина и мага, а для всех остальных общую

```
func void DIA_SylvioDJG_HelloAgain_Info()
{
	if((other.guild == GIL_SLD) || (other.guild == GIL_DJG))
	{
		AI_Output(self,other,"DIA_SylvioDJG_HelloAgain_09_00");	//Ох, будь я проклят! Ты тоже решил подзаработать, ха? Я знал это! У тебя душа наемника.
		AI_Output(self,other,"DIA_SylvioDJG_HelloAgain_09_01");	//Послушай, если ты думаешь, что загребешь здесь кучу золота, ты жестоко ошибаешься. Я был здесь первым.
	}
	else if(other.guild == GIL_KDF)
	{
		AI_Output(self,other,"DIA_SylvioDJG_HelloAgain_09_02");	//Эй, ты! Меня тошнит от магов! Иди куда-нибудь еще! Здесь для тебя ничего нет.
	}
	else
	{
		AI_Output(self,other,"DIA_SylvioDJG_HelloAgain_09_03");	//Эй ты, паладин! Возвращайся в свою шахту. Здесь для тебя ничего нет.
	};
	AI_Output(other,self,"DIA_SylvioDJG_HelloAgain_15_04");	//Понял. Никто не хочет расставаться с награбленным.
	AI_Output(self,other,"DIA_SylvioDJG_HelloAgain_09_05");	//Ты сам сказал это. Исчезни.
};
```

**Comment by dosinabox** [comment]
> диалог Сильвио надо переделать. проверку сделать на фразу для паладина и мага, а для всех остальных общую

сделано

> Жан в оригинале тоже делает доспехи нейтралу?

Доспехи только для ОнД. А для всех остальных золото за кровь.

**Comment by Kor-Angar** [comment]
надо Жана править. Сейчас он у нейтрала берёт чешую, делает доспехи, но не продаёт. [или что бы продавал](https://github.com/Kor-Angar/G2_NotR_Kor_Angar/commit/c2a6f5f2c76b3dc6a8d1bfc7d96686acb5124d50) или что бы не забирал чешую для доспехов. то что сейчас это гоп стоп :-)

**Comment by Kor-Angar** [comment]
тут квест запускается 

```
func void DIA_Jan_SellArmor_Info()
{
	AI_Output(other,self,"DIA_Jan_SellArmor_15_00");	//Могу я купить у тебя доспехи?
	if(hero.guild == GIL_PAL)
	{
		AI_Output(self,other,"DIA_Jan_SellArmor_10_01");	//Мои доспехи не идут ни в какое сравнение с теми, что у тебя сейчас есть. Забудь об этом.
	}
	else if(hero.guild == GIL_KDF)
	{
		AI_Output(self,other,"DIA_Jan_SellArmor_10_02");	//Маг, который нуждается в доспехах? Забудь об этом - тебе лучше обратиться к портному. Я не могу сделать доспехи для тебя.
	}
	else
	{
		AI_Output(self,other,"DIA_Jan_SellArmor_10_03");	//Если бы они у меня были, я бы продал их тебе. Но, к сожалению, у меня их нет.
		Info_ClearChoices(DIA_JAN_SellArmor);
		Info_AddChoice(DIA_JAN_SellArmor,"Думаю, ничего не получится.",DIA_JAN_SellArmor_Sorry);
		Info_AddChoice(DIA_JAN_SellArmor,"Ты можешь изготовить что-нибудь для меня?",DIA_JAN_SellArmor_BuildOne);
	};
};
```

**Comment by Kor-Angar** [comment]
квест про судью наверное тоже надо дать нейтралу

**Comment by Kor-Angar** [comment]
у Дневника в тайной библиотеки стоит проверка на гильдию ОНД, а у Альманаха нет. вбил тексты в табличку
https://steamcommunity.com/sharedfiles/filedetails/?id=2255221718

**Comment by Kor-Angar** [comment]
в этот раз я не смог ничего придумать. поэтому вернулся к нейтральному варианту в виде старых доспехов паладина и двуручного меча ОнД. тексты взял из Паладина и ОнД 
```
Doc_PrintLines(nDocID,0,"... Чем они сильнее, тем, похоже, для него выше их ценность как слуг-нежити. Эти обращенные паладины очень сильны, и с ними очень тяжело справиться. Один из них попал в мои руки. Мне остается только надеяться, что другие не почувствуют его присутствие здесь ...");
Doc_SetMargins(nDocID,-1,30,20,275,20,1);
Doc_SetFont(nDocID,1,FONT_BookHeadline);
Doc_SetFont(nDocID,1,FONT_Book);
Doc_PrintLines(nDocID,1,"... Обращенный паладин, похоже, ни на что не реагирует. Я запер его доспехи и остальные его вещи в задней комнате. Дверь в нее можно открыть только изнутри. При помощи  руны можно телепортироваться в эту комнату. Для её создания необходим чистый рунный камень и небольшая бутылочка святой воды ...");
```

```
Doc_PrintLines(nDocID,1,"Оружие повелителя драконов.");
Doc_PrintLine(nDocID,1,"");
Doc_PrintLines(nDocID,1,"Чтобы изготовить меч, достойный повелителя драконов, этот меч должен быть закален в драконьей крови. 5 мензурок крови могут придать стали непревзойденную остроту и прочность.");
Doc_PrintLine(nDocID,1,"");
Doc_PrintLines(nDocID,1,"Теперь я вполне уверен, что слезы Инноса - именно то, что войны Инноса использовали в давно забытом ритуале 'Освящение Меча'. Это означает, что при помощи этой бутылочки можно придать дополнительную силу оружию.");
Doc_Show(nDocID);
//PlayergetsFinalDJGArmor = TRUE;
```
https://github.com/Kor-Angar/G2_NotR_Kor_Angar/commit/bf84a7aa92344f5877619969e12ac98f7077c553
может быть в следующий раз придет понимание.

**Comment by dosinabox** [comment]
> что то Рози не пошла за героем в 5 главе

Некуда)

```
if((hero.guild == GIL_MIL) || (hero.guild == GIL_PAL))
{
	AI_Output(other,self,"DIA_Rosi_HILFE_15_01");	//Я могу отвести тебя в город.
}
else if((hero.guild == GIL_SLD) || (hero.guild == GIL_DJG))
{
	AI_Output(other,self,"DIA_Rosi_HILFE_15_02");	//Я отведу тебя на ферму лендлорда.
}
else if(hero.guild == GIL_KDF)
{
	AI_Output(other,self,"DIA_Rosi_HILFE_15_03");	//Пойдем в монастырь. Тебя хорошо примут там.
};

...

if((hero.guild == GIL_MIL) || (hero.guild == GIL_PAL))
{
	Npc_ExchangeRoutine(self,"FollowCity");
	B_StartOtherRoutine(Till,"FollowCity");
}
else if((hero.guild == GIL_SLD) || (hero.guild == GIL_DJG))
{
	Npc_ExchangeRoutine(self,"FollowBigfarm");
	B_StartOtherRoutine(Till,"FollowBigfarm");
}
else if(hero.guild == GIL_KDF)
{
	Npc_ExchangeRoutine(self,"FollowKloster");
	B_StartOtherRoutine(Till,"FollowKloster");
};
```

**Comment by Kor-Angar** [comment]
надо к единообразию стараться приводить везде. типа

if((hero.guild == GIL_MIL) || (hero.guild == GIL_PAL))
{

}
else if (hero.guild == GIL_KDF)
{

}
else 
{

};

**Comment by Kor-Angar** [comment]
мне так кажется :-)

**Comment by dosinabox** [comment]
Куда должен отводить Рози нейтрал?

**Comment by Kor-Angar** [comment]
дай ему выбор или на ферму или в город

**Comment by Kor-Angar** [comment]
хотя нейтрал может и в монастырь отвести

**Comment by Kor-Angar** [comment]
наверное Ли должен давать задание нейтралу про Горна, так как нейтрал к Хагену ходил через Ли
https://github.com/Kor-Angar/G2_NotR_Kor_Angar/commit/1d991667e4530b0104853c7b70cd079b185dc51e

**Comment by Kor-Angar** [comment]
тут https://github.com/dosinabox/g2nr_unofficial_update/blob/7f3f46065207609f9d794dd37a01200104199ce0/PrjGOTHIC/Story/B_Story/B_Enter_NewWorld.d#L137

```D
if((hero.guild == GIL_MIL) || (hero.guild == GIL_PAL))
		{
			Wld_InsertItem(ItAm_Dex_01,"FP_ROAM_XARDAS_SECRET_26");
		}
		else if((hero.guild == GIL_SLD) || (hero.guild == GIL_DJG))
		{
			Wld_InsertItem(ItAm_Hp_01,"FP_ROAM_XARDAS_SECRET_26");
		}
		else
		{
			Wld_InsertItem(ItAm_Hp_Mana_01,"FP_ROAM_XARDAS_SECRET_26");
		};
```
поменять надо на
проверка на мага / послушника ItAm_Hp_Mana_01
проверка на ментра / пала ItAm_Dex_01
прочие ItAm_Hp_01

**Comment by dosinabox** [comment]
готово

**Comment by Kor-Angar** [comment]
3 глава. Беннет в тюрьме. проверить диалоги для нейтрала.

**Comment by Kor-Angar** [comment]
```D
instance DIA_Bennet_WannaJoin(C_Info)
{
	npc = SLD_809_Bennet;
	nr = 4;
	condition = DIA_Bennet_WannaJoin_Condition;
	information = DIA_Bennet_WannaJoin_Info;
	permanent = TRUE;
	description = "Я хочу присоединиться к наемникам!";
};


func int DIA_Bennet_WannaJoin_Condition()
{
	if(Npc_KnowsInfo(other,DIA_Bennet_BauOrSld) && (other.guild == GIL_NONE))
	{
		return TRUE;
	};
};

```

**Comment by Kor-Angar** [comment]
Ян

**Reply by Kor-Angar** [reply]
в DIA_Jan_Dragonscales_Condition добавить гильдию ОнД

**Reply by Kor-Angar** [reply]
в dia_jan_sellarmor_info добавить проверку на гильдию ОнД

**Reply by dosinabox** [reply]
Диалог DIA_Jan_SellArmor_Info содержит проверки гильдии и разные реплики. Нужно редактировать диалог, а не его условие:
```cpp
func void DIA_Jan_SellArmor_Info()
{
	AI_Output(other,self,"DIA_Jan_SellArmor_15_00");	//Могу я купить у тебя доспехи?
	if(hero.guild == GIL_PAL)
	{
		AI_Output(self,other,"DIA_Jan_SellArmor_10_01");	//Мои доспехи не идут ни в какое сравнение с теми, что у тебя сейчас есть. Забудь об этом.
	}
	else if(hero.guild == GIL_KDF)
	{
		AI_Output(self,other,"DIA_Jan_SellArmor_10_02");	//Маг, который нуждается в доспехах? Забудь об этом - тебе лучше обратиться к портному. Я не могу сделать доспехи для тебя.
	}
	else if(hero.guild == GIL_DJG)
	{
		AI_Output(self,other,"DIA_Jan_SellArmor_10_03");	//Если бы они у меня были, я бы продал их тебе. Но, к сожалению, у меня их нет.
		Info_ClearChoices(DIA_JAN_SellArmor);
		Info_AddChoice(DIA_JAN_SellArmor,"Думаю, ничего не получится.",DIA_JAN_SellArmor_Sorry);
		Info_AddChoice(DIA_JAN_SellArmor,"Ты можешь изготовить что-нибудь для меня?",DIA_JAN_SellArmor_BuildOne);
	}
	else
	{
		AI_Output(self,other,"DIA_Jan_SellArmor_10_01_add");	//Забудь об этом.
	};
};
```

**Reply by dosinabox** [reply]
Тогда и выход на DIA_Jan_Dragonscales будет только у ОНД.

**Reply by Kor-Angar** [reply]
> Нужно редактировать диалог, а не его условие

я же про него и написал :-) что его надо редактировать.

**Reply by dosinabox** [reply]
готово, DIA_Jan_Dragonscales_Condition трогать не нужно

**Reply by Kor-Angar** [reply]
я при прохождение добавлял в DIA_Jan_Dragonscales_Condition доп условия  так как у меня уже маркер был получен DIA_JAN_SellArmor_permanent

**Comment by dosinabox** [comment]
Финальный ролик с драконом без ГГ (из мода Предопределение):
[Extro_NONE.zip](https://github.com/dosinabox/g2nr_unofficial_update/files/13906690/Extro_NONE.zip)

**Comment by Kor-Angar** [comment]
Герою нейтралу можно дать возможность продавать чешую Беннету. в принципе можно такую возможность дать всем фракциям.

**Reply by Kor-Angar** [reply]
я делал отдельный инфо для диалога что бы не переписывать основной. в твоём проекте можно просто доп условия сделать в основной инфо функции.

```D
func int dia_bennet_dragonscale_condition()
{
	if (Npc_HasItems(other,itat_dragonscale) > 0) 
    {
        if (hero.guild == GIL_DJG)
        {
            DIA_BENNET_DRAGONSCALE.information = dia_bennet_dragonscale_info;
            return TRUE;
        };
        
        DIA_BENNET_DRAGONSCALE.information = dia_bennet_dragonscaleGilNone_info;

        return TRUE;

    };
    
};

func void dia_bennet_dragonscaleGilNone_info()
{
	AI_Output(other,self,"DIA_Bennet_DragonScale_15_00");	//Я принес тебе несколько драконьих чешуек.
	AI_Output(self,other,"DIA_Bennet_DragonScale_06_01");	//Настоящая чешуя дракона!
	AI_Output(self,other,"DIA_Bennet_DragonScale_06_02");	//Вот твое золото!
	//BENNET_DRAGONSCALE_COUNTER = BENNET_DRAGONSCALE_COUNTER + Npc_HasItems(other,itat_dragonscale);
	b_giveinvitems(self,other,itmi_gold,Npc_HasItems(other,itat_dragonscale) * VALUE_DRAGONSCALE);
	b_giveinvitems(other,self,itat_dragonscale,Npc_HasItems(other,itat_dragonscale));
    /*
	if((BENNET_DRAGONSCALE_COUNTER >= 20) && (SHOW_DJG_ARMOR_M == FALSE))
	{
		AI_Output(self,other,"DIA_Bennet_DragonScale_06_03");	//Хорошо, этого должно быть достаточно. Я могу продать тебе новые, улучшенные доспехи, если, конечно, тебе это интересно.
		SHOW_DJG_ARMOR_M = TRUE;
	};
    */
};
```

**Comment by Kor-Angar** [comment]
так же нейтралу я хотел дать следующие квесты в третей главе:

1) квест про траву Андре, при условию что ликвидирована гильдия воров.
2) квест на поддельные акции который выдаёт верховный маг огня 
3) квесты судьи (для мирного захвата корабля в 5 главе)
